alicloud-test/tests/alicloud_cms_monitor_host/test-get-query.sql

```diff
@@ -0,0 +1,3 @@
+select host_name, instance_id, title, region
+from alicloud_cms_monitor_host
+where host_name = '{{ output.host_name.value }}' and instance_id = '{{ output.instance_id.value }}'
```

Comment:
key column in table is host_name then why we are checking for instance_id as well.